packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -24,19 +25,29 @@ const AppContents = ({
     is_route_modal_on,
     notifyAppInstall,
     platform,
-    pageView,
     pushDataLayer,
     setAppContentsScrollRef,
 }) => {
     const [show_cookie_banner, setShowCookieBanner] = React.useState(false);
     const [is_gtm_tracking, setIsGtmTracking] = React.useState(false);
     const { is_appstore } = React.useContext(PlatformContext);
+    const {
+        client: { user_id },
+    } = useStore();
 
     const tracking_status = tracking_status_cookie.get(TRACKING_STATUS_KEY);
 
     const scroll_ref = React.useRef(null);
 
     React.useEffect(() => {
+        // rudderstack page view trigger
+        WS.wait('authorize').then(() => {
+            rudderstack.identifyEvent(user_id, {
+                language: getLanguage().toLowerCase() || 'en',
+            });
+            rudderstack.pageView();
```

Comment:
why do we need to call `pageView()` again? because It's being called inside `identyfyEvent` :thinking:

Reply:
and correct me if I'm wrong, we are calling `identyfyEvent` inside `client-store` after user is logged in. so here when we call it again, it's not going to do anything because `has_initialized` is already set to true , right?

Reply:
Actually I've added `rudderstack.reset()` when the user logs out, so this resets the state `has_identified` to `false`

Reply:
Actually we don't need the `pageView()` here as well, I placed it there since it was in the old rudderstack store as well, I'll remove it 😅